src/resolve/resolved-dns-stub.c

```diff
@@ -502,19 +502,21 @@ static int set_dns_stub_common_socket_options(int fd, int family) {
         return 0;
 }
 
-static int manager_dns_stub_udp_fd(Manager *m) {
+static int manager_dns_stub_fd(Manager *m, bool udp) {
```

Comment:
why not pass "int type" here, that is SOCK_DRGAM or SOCK_STREAM. Would be much nicer I think